test/utils/initDB.js

```diff
@@ -74,3 +74,22 @@ client.hmsetAsync("event:400", "eventName", pollToConfirmed.eventName,
 
 // client.saddAsync("notifications:12345678", JSON.stringify(fixtures.pollToConfirmedNotification));
 // client.sadd("calendar:" + fixtures.HARRY_ID, fixtures.pollToConfirmedEventID);
+
+
+
+/*
+SET DEFAULT CONFIRMED EVENT 500
```

Comment:
There's technically already a default confirmed event, I believe event:100.  No worries if you want a separate event reserved specifically for the deletion test, but then reference it as such to avoid confusion